docs/conf.py

```diff
@@ -29,7 +32,16 @@
 project = u'icalendar'
 this_year = datetime.date.today().year
 copyright = u'{}, Plone Foundation'.format(this_year)
-version = pkg_resources.get_distribution('icalendar').version
+
+# Read the version number
+with open(os.path.join(base_dir, 'src/icalendar/__init__.py')) as f:
+    for line in f.readlines():
+        if line.startswith('__version__'):
+            version_info = {}
+            exec(line, None, version_info)
```

Comment:
I dislike using ``exec`` here.
Please refactor to do it like here: https://github.com/collective/icalendar/blob/master/setup.py#L7